src/createServer.js

```diff
@@ -1,3 +1,38 @@
 // Write code here
 // Also, you can create additional files in the src folder
 // and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase/convertToCase');
+const { errorHandler } = require('./requestHandlers/errorHandler');
+
+function createServer() {
+  const server = http.createServer(async(req, res) => {
+    res.setHeader('Content-type', 'application/json');
+
+    const normalizedURL = new URL(req.url, 'http://localhost:5700');
+
+    const originalText = normalizedURL.pathname.slice(1);
+    const targetCase = normalizedURL.searchParams.get('toCase');
+
```

Comment:
Here you can make validation if data which comes from request is valid and if not you can return an error. Else you can use convertedToCase func.